docs/source/tutorials/ATL03/IceSat_ATL03_Access.ipynb

```diff
@@ -0,0 +1,781 @@
+{
```

Comment:
<p>Recommend:</p><p>Read H5 file to... -&gt; Read the H5 file to...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>